persist/fs/files.go

```diff
@@ -269,6 +306,26 @@ func findFiles(fileDir string, pattern string, fn toSortableFn) ([]string, error
 	return matched, nil
 }
 
+type directoryNamesToPaths map[string]string
+
+func findSubDirectoriesAndPaths(directoryPath string) (directoryNamesToPaths, error) {
+	parent, err := os.Open(directoryPath)
+	if err != nil {
+		return nil, err
+	}
+	defer parent.Close()
```

Comment:
iirc `Close()` returns an error, could you check it instead of dropping silently

Reply:
ahh thanks!

Reply:
+1